### ISSUE TITLE
Setting a logLevel from Virtuoso results in a not responding Server

### ISSUE THREAD

**Issue by TM90** [unihd-cag/skillbridge#56]
pyStartServer with any logLevel even "WARNING" results in the server not responding
pyStartServer without the logLevel argument works.

**Comment by nielsbuwen** [comment]
The logLevel "DEBUG" definitely works:

```
pyStartServer "default" "DEBUG"
```

**Comment by TM90** [comment]
Using it like this in the skill console produces the error pyStartServer(logLevel="DEBUG")

**Comment by nielsbuwen** [comment]
Looks like you cannot use optional parameters like this

```
defun(test (@optional (x 1) (y 2))
    printf("x=%L y=%L\n" x y)
)

test(y=20)
```

Prints "x=20 y=2"

**Comment by TM90** [comment]
Ok then we should update the documentation.

**Comment by TM90** [comment]
@key has the intended behaviour:

```
defun(test (@key (x 1) (y 2))
	printf("x=%L y=%L\n" x y)
)

test
>>> x=1 y=2
test(?y 10)
>>> x=1 y=10
```

**Comment by nielsbuwen** [comment]
Closed with #57
